modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -54,4 +54,4 @@ image::op-pipeline-builder-task-details.png[]
 ... In the *Output Resources* -> *Image* section, click the *Select Resources* list, and select *Img*.
 ... If required, in the *Parameters* section, add more parameters to the default ones, by using the `$(params.<param-name>)` syntax.
 ... Similarly, add an input resource for the *openshift-client* task.
-. Click *Create* to create and view the pipeline in the *Pipeline Details* and the click the *Action* button to start the pipeline.
+. Click *Create* to create and view the pipeline in the *Pipeline Details* page, and then click the *Action* button to start the pipeline.
```

Comment:
Adding to what Karthik suggested
```suggestion
. Click *Create* to create and view the pipeline in the *Pipeline Details* page, and then click on the *Action* button to start the pipeline.
```

Reply:
Done. @sanketpathak I have changed pipeline to Pipeline. It is Actions drop-down. I have applied Karthik's comments. We do not say click on, so ignoring that comment. Thanks!